app/views/simulator/edit.html.erb

```diff
@@ -622,8 +622,21 @@
   });
 </script>
 
-<link rel="stylesheet" type="text/css" href="/css/restrictedElements.css">
+<!-- on click for dropdowns -->
+<script>
+$('.dropdown-caret').click(function(){
```

Comment:
You don't need javascript to do this,  you can just use plain css by checking if parent dom object is having 'show' class